### PR TITLE
Correct the source/target compatibility property usage

### PR DESCRIPTION
**Pull request by littleaj**
The new version of gradle had some issue with the `tasks.withType` method. This seemed to work just fine.

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -36,9 +36,9 @@ if (System.env.'COLLECTD_HOME') {
 //include 'distributions'
 
 
+include ':test:webapps:PerfTestApp'
```
Comment (trask) [inline settings.gradle]: i thought this was sample app?

File: test/fakeIngestion/servlet/build.gradle [hunk 2]
```diff
@@ -1,7 +1,7 @@
+apply plugin: 'java'
 apply plugin: 'war'
 
 repositories {
-    jcenter()
```
Comment (trask) [inline test/fakeIngestion/servlet/build.gradle]: 👍

File: test/webapps/PerfTestApp/build.gradle [hunk 3]
```diff
@@ -3,25 +3,18 @@ apply plugin: 'war'
 def aiSdkVersion = version
 
 dependencies {
-    compile "com.microsoft.azure:applicationinsights-core:$aiSdkVersion"
+    compile project(':core')
```
Comment (trask) [inline test/webapps/PerfTestApp/build.gradle]: 👍

**Comment by trask** [comment]
Also looks like some issues with build

File: settings.gradle (hunk 1 above)
Comment (littleaj) [reply settings.gradle]: oops, I didn't mean to include that...will revert. I was using that application for testing.

**Comment by littleaj** [comment]
@trask, build issues were because of some compatibility issues between `:test:smoke:framework:testCases` and `:test:webapps:PerfTestApp`. #995 fixes this. The inclusion of the PerfTestApp project in settings.xml was unintentional and since it's reverted, that should address the build issues.

**Comment by littleaj** [comment]
I think #995 needs to be merged first. That's how I was running locally.

**Comment by littleaj** [comment]
ok. Fixed a few more compatibility issues. Just waiting on builds to pass

**Pull request merged**